sql/moz-fx-data-shared-prod/telemetry_dev_cycle_derived/firefox_major_release_dates_v1/query.sql

```diff
@@ -0,0 +1,17 @@
+WITH firefox_releases AS (
+  SELECT
+    CAST(REGEXP_EXTRACT(build.target.version, r"^\d+") AS INT64) AS version,
+    build.target.channel,
+    DATE(MIN(build.download.date)) AS publish_date
+  FROM
+    `mozdata.telemetry.buildhub2`
```

Comment:
```suggestion
    telemetry.buildhub2
```

Reply:
but you can't leave the mozdata out .. it's running in `moz-fx-data-shared-prod`?

Reply:
should also be in moz-prod